t/mojo-oauth2.t

```diff
@@ -13,6 +13,7 @@ use lib qw( . lib local/lib/perl5 );
 BEGIN {
   $ENV{LOG4PERL_CONFIG_FILE}     = 'log4perl-t.conf';
   $ENV{BUGZILLA_DISABLE_HOSTAGE} = 1;
+  $ENV{MOJO_TEST}                = 1;
```

Comment:
This is similar to `BUGZILLA_UNSAFE_AUTH_DELEGATION`, I wonder if shouldn't rename both to something like `BUGZILLA_ALLOW_UNSAFE_HTTP`?

Reply:
s/UNSAFE/INSECURE/

Reply:
Problem is since we are using Test::Mojo, we are just passing routes in the test suite and not fully qualified URLs. So we can't even check for any schemes from what I can tell.